src/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/WrappedGr.cpp

```diff
@@ -9,39 +9,14 @@
 #include "PointwiseFunctions/AnalyticSolutions/GeneralRelativity/GaugeWave.hpp"
 #include "PointwiseFunctions/AnalyticSolutions/GeneralRelativity/KerrSchild.hpp"  // IWYU pragma: keep
 #include "PointwiseFunctions/AnalyticSolutions/GeneralRelativity/Minkowski.hpp"  // IWYU pragma: keep
+#include "PointwiseFunctions/AnalyticSolutions/GrMhd/BondiMichel.hpp"
```

Comment:
I didn't realize this until just now (sorry!) but this means the GR analytic solutions library needs a link dependency on RelEuler and GRMHD. I _think_ this creates a cyclic dependency with the TOV solution. I'm not immediately sure what the best solution is, but one option could be to add a `tpp` for `WrappedGr`, instantiate the vacuum ones here, and the vacuum+hydro ones in the respective hydro directories since those already depend on the GH solutions. Thoughts?